src/components/VBreadcrumbs/VBreadcrumbs.vue

```diff
@@ -38,11 +39,11 @@ export default {
   },
 
   computed: {
-    routeBreadcrumbs() {
-      if (this.breadcrumbs.length) {
+    routeBreadcrumbs({ $route }) {
+      if (this.breadcrumbs['length']) {
```

Comment:
Was this just an auto formatting change?

Reply:
Correct

Reply:
Partially correct, to be more specific, Vetur, was screaming at me and seemed to like this format.